From 13c863cb197ce5fe3490c497fc623e44f9f6d6d4 Mon Sep 17 00:00:00 2001
From: Ryan Brue <ryanbrue.dev@gmail.com>
Date: Tue, 12 Nov 2024 14:40:41 -0600
Subject: [PATCH] fix: don't set XCURSOR_THEME in start-cosmic

Setting XCURSOR_THEME messes up custom cursor theme overrides, so let's set the Cosmic cursor theme another way in distros and such

Signed-off-by: Ryan Brue <ryanbrue.dev@gmail.com>
---
 data/start-cosmic | 1 -
 1 file changed, 1 deletion(-)

diff --git a/data/start-cosmic b/data/start-cosmic
index 323f271..81d0d57 100755
--- a/data/start-cosmic
+++ b/data/start-cosmic
@@ -30,7 +30,6 @@ fi
 
 export XDG_CURRENT_DESKTOP="${XDG_CURRENT_DESKTOP:=COSMIC}"
 export XDG_SESSION_TYPE="${XDG_SESSION_TYPE:=wayland}"
-export XCURSOR_THEME="${XCURSOR_THEME:=Cosmic}"
 export _JAVA_AWT_WM_NONREPARENTING=1
 export GDK_BACKEND=wayland,x11
 export MOZ_ENABLE_WAYLAND=1
